ci/datadog-synthetics.yml

```diff
@@ -0,0 +1,29 @@
+# This workflow uses actions that are not certified by GitHub.
+# They are provided by a third-party and are governed by
+# separate terms of service, privacy policy, and support
+# documentation.
+
+name: Run Datadog Synthetic tests
+
+on:
+  push:
+    branches: [ $default-branch ]
+  pull_request:
+    branches: [ $default-branch ]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v2
+
+    # Run Synthetic tests within your GitHub workflow.
+    # For additional configuration options visit the action within the marketplace: https://github.com/marketplace/actions/datadog-synthetics-ci
+    - name: Run Datadog Synthetic tests
+      uses: DataDog/synthetics-ci-github-action@v0.2.0
+      with:
+        api_key: ${{secrets.DD_API_KEY}}
+        app_key: ${{secrets.DD_APP_KEY}}
+        test_search_query: 'tag:e2e-tests' #Modify this tag to suit your tagging strategy
```

Comment:
Does the action look for specific file in the repo? Looks like [so](https://github.com/DataDog/synthetics-ci-github-action#example-workflow-using-an-existing-syntheticsjson-file). If yes, then you may consider enhancing our internal detection logic for such a file. And that will help in pushing this workflow up in order in the UI.
Let me know if this is applicable and I can provide more details.

Reply:
BTW this should block the current PR.

Reply:
@bishal-pdMSFT when specific params aren't passed to the action we do fall back to look for a file. If you could provide additional information on the internal detection logic that would be great.